### ISSUE TITLE
Misleading data file icon in catalog view

### ISSUE THREAD

**Issue by jagoldstein** [NCEAS/metacatui#921]
@mpsaloha pointed out that he was misled by search results when trying to find a social science dataset that contained data objects. This example https://arcticdata.io/catalog/view/urn:uuid:2d6da69f-606e-4c14-971b-58c05e560fb2 displays the "This dataset contains data files" icon in the search results on the catalog page view, as shown here:

<img width="353" alt="screenshot2019-02-27" src="https://user-images.githubusercontent.com/18270215/53532696-b65f5e00-3aac-11e9-9eca-7f26bd151c15.png">

But, the only files contained are 4 child EMLs, and each child package is also devoid of data objects. The "data files [are included]" icon (shown here)

<img width="35" alt="data_icon" src="https://user-images.githubusercontent.com/18270215/53532880-819fd680-3aad-11e9-9aea-f91f9aae3876.png">

should not be displayed in search results on the catalog page view unless there are *data* objects in the package; metadata files should not qualify.

**Comment by laurenwalker** [comment]
Unfortunately there is no way to know if the objects are data or metadata without sending an additional query to Solr, which slows down the UI. That icon and message are displayed by simply counting the number of pids that EML document "documents" via Solr. That resource map must have RDF statements that state the parent EML documents the other 4 EMLs. 

I'll keep this issue open so we can hopefully find a way to fix this issue, but hopefully it is a rare occurrence until then.

**Comment by jagoldstein** [comment]
@mpsaloha 

Thanks Lauren. With nesting, isn't the typical structure that the parent's RDF "aggregates" each child's _RDF_ PID, not the child EMLs?

**Comment by laurenwalker** [comment]
Yes, that is how nested packages work. You had mentioned in the original issue description that the package was aggregating 4 child EMLs, so that is what I was referring to above.

**Comment by laurenwalker** [comment]
Ok. It ultimately doesn't matter what kind of object it aggregates - the number shown with the icon is just counting the number of `documents` statements from the package.
